[vividus-plugin-rest-api] Add possibility to validate response archive with comparison rule (#2711)
vividus-framework/vividus: docs/modules/plugins/pages/plugin-rest-api.adoc

=== Validate archive from response body

Verifies that at least one (or no one) entry in a response archive matches the specified xref:parameters:string-comparison-rule.adoc[string comparison rule].

> If column with comparison rule is not exists, verifies that one of archive entries has one of the specified names.

Reviewer: ```suggestion
If comparison rule column does not exists, the verification that archive entries have the specified names is performed.
```

[source,gherkin]
----
Then the response archive contains entries with the names:$parameters

> ----

Reviewer 2: ```
Then the response archive contains entries with the names:$parameters
```

Refactor wording to:
```
Then response archive contains entries with names:$parameters
```
